gen-bundle-image.sh

```diff
@@ -8,16 +8,16 @@ function set_bundle_variables {
    local version=$1
    local preset=$2
 
-   if [[ ${preset} = "podman" ]]; then
-    vfkit_bundle_arm64=crc_podman_vfkit_${version}_arm64.crcbundle
-   fi
-
    if [[ ${preset} = "openshift" ]]; then
        preset=""
    else
        preset="_${preset}"
    fi
 
+   if [ ${preset} = "_podman" -o ${preset} = "" ]; then
+       vfkit_bundle_arm64=crc${preset}_vfkit_${version}_arm64.crcbundle
+   fi
+
```

Comment:
If you keep this in the same place as the `podman` check was, this is more readable, no test on an empty preset, no need to prepend `_`